packages/experimental/Divider/src/DividerTokens.ios.ts

```diff
@@ -0,0 +1,11 @@
+import { buildUseTokens } from '@fluentui-react-native/framework';
+import type { Theme } from '@fluentui-react-native/framework';
+import type { DividerTokens } from './Divider.types';
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
+
+export const useDividerTokens = buildUseTokens<DividerTokens>((theme: Theme) => ({
+  // base tokens
+  insetSize: 0,
+  lineColor: theme.colors.neutralStroke2,
+  thickness: globalTokens.stroke.width05,
```

Comment:
This is from iOS designs? I would understand that we are not calling the iOS component completed unless the team validates back on designs and implementation - is that a fair statement?

Reply:
Correct. I have the behaviour and tokens files added but iOS engineers needs to validate on there end to mark it complete on iOS + Adding the E2E tests.